web/src/helpers/language.js

```diff
@@ -0,0 +1,7 @@
+import { getKey } from '../helpers/storage';
+import { LANGUAGE_NAMES } from '../helpers/constants';
+
+export const getPreferredLanguage = () => {
+  const preferredLanguage = getKey('i18nextLng');
```

Comment:
Is it maybe better to use the built in function [here](https://www.i18next.com/overview/api#language) incase they change their implementation and the local storage value isn't there?

Reply:
I'd also be a fan of using the actual language key (from i18n) instead of the typed out language name. For example (en-GB) for British English.
It should be more robust if we or i18n change anything in the future (We might want to show the language in both the selected language and native language).
However this might require some bigger changes.